clean_newsarticles.py

```diff
@@ -0,0 +1,73 @@
+
+import re
+import csv
+
+#input_file = r'data\novinove_clanky\Fake News Detection\fake.csv'
+
+#pattern pro webove stranky https a twitter musi jit out
+
+
+def read_csv_to_dict(input_file):
+    """Reads a CSV file and returns the data as a list."""
+    content_list = [] 
+    try: 
+        with open(input_file, 'r', encoding='utf-8') as f:
+            reader = csv.reader(f) 
+            for raw in reader:  
+                    content_list.append(raw)
+        del content_list[-3:]

```

Comment:
Please document this line 'del content_list[-3:]'. It removes any last 3 items with no condition. It may be not usable for any other documents. It doesn't adapt to any other text.